### DISCUSSION TITLE
Error Capstone Flask-UI

### DISCUSSION THREAD

**Discussion by d2salit** [teamalgoritma/community#862] [Q&A]
Mba mas,

Saya kesulitan di coding Task 7, membuat par plot

![image](https://github.com/teamalgoritma/community/assets/142375570/50ad2e36-749e-4fdb-9a88-4965f68788f3)

apakah coding-an saya ini benar?

Terima kasih.

**Comment by d2salit** [comment]
sudah solved. Terima kasih.

**Reply by ranydwi** [reply]
Dear @d2salit,

Glad to know that your obstacle has been cleared. 
Sebetulnya dari code Ibu terdapat missed memanggil kolom untuk x dan y berdasarkan data `cat_order`, sehingga tidak muncul tampilan di visualisasinya.

Regards,
Team Algoritma
